.devcontainer/devcontainer.json

```diff
@@ -35,5 +36,6 @@
         "davidanson.vscode-markdownlint",
         "ms-azuretools.vscode-docker",
         "vscode-icons-team.vscode-icons",
+        "github.copilot",
```

Comment:
well no.

while i do like copilot personally - that's clearly too controversial to force onto future developers by default.
everyone can install this on his own within the container - but it should clearly not be part of the default config - at least not at this point.

Reply:
Yes your are right. This one I just forgot to clean up, when I was preparing the PR